locales/en/index.yml

```diff
@@ -2537,6 +2537,8 @@ bonus:
         title: "Do you really want to delete the following methods from the Wallet?"
         subtitle: "You can add them again at any time, but you will have to re-enter the data and repeat the authorization procedures."
   cgn:
+    generic:
+      linkError: "The url could not be opened at the moment, please try again"
```

Comment:
@AliceAzzolini we need your love here

Reply:
"The site can't be reached at the moment. Please, try again later."